docs/src/content/components/text-caps.mdx

```diff
@@ -0,0 +1,92 @@
+---
+name: Caps
+route: /components/text-caps
+---
+
+import { Caps } from '@pluralsight/ps-design-system-text'
+
+# Caps
+
+<Intro>Caps are similar to Label, but all uppercase</Intro>
```

Comment:
If the distinguishing feature between Caps and Label is the casing, I wonder why they are not closer aligned in the system.  This could be through naming (eg, `CapsLabel`) or even just properties (eg, `<Label appearance="caps" />`).

Reply:
Because Caps uses smaller px sizes for the small, medium, large there isn't a 1:1 relationship with Label.

Reply:
👍 Sounds good.

If, however, we called in `CapsLabel` or used `appearance="caps"`, that could remain true.

I was just exploring the relationship between the two: "If they're so close beyond visual style, why are they separate?", kind of thing.

Reply:
I removed the Caps component and added a boolean prop called caps to the Label component.

Reply:
Oh, cool.  How'd it go?  I'd be interested to hear any of your thinking around why you made the change.

Reply:
I ended up with one extra caps size than necessary, but it's probably simpler approach than a separate Caps component. One adjustment I'd like to make is for the `<Labels caps>` font-sizes to be a bit smaller than their non-caps counterparts.

Reply:
@jaketrent I removed the caps component and added two of the smallest caps styles to heading since that is typically how they are used.